stylix/testbed/modules/application.nix

```
@@ -84,19 +94,11 @@ in
   };
 
   config = lib.mkIf (config.stylix.testbed.ui != null) {
-    services = {
-      displayManager.gdm.enable = true;
-      desktopManager.gnome.enable = true;
-    };
-
     services.displayManager.autoLogin = {
-      enable = true;
+      enable = lib.mkDefault true;
```

I'd rather use mkForce when disabling this then mkDefault when enabling it
```suggestion
      enable = true;
```

---

Done, but can you explain your rationale?